* changes to deal with synsets.json from mill export command

hmuniz opened the pull request:
- update synset struct
  - update poses

hmuniz committed:
* changes to deals with synsets.json from mill export command
  - update synset struct
  - update poses

odanoburu commented on sensetion-data.el:
```diff
@@ -123,9 +123,9 @@ Slots:
+  (concat (sensetion--synset-lexname synset)
           "-"
-          (sensetion--synset-pos synset)))
+          (car (sensetion--synset-terms synset))))
```
lexname sem pos + lexical form é ambíguo, mas usar o primeiro sensekey também não me parece muito readable.. será que vale usarmos o esquema de sense key simplificada (pos, lexname, lexform, lexid)?

odanoburu commented on sensetion.el:
```diff
@@ -609,7 +609,7 @@ gloss."
 		 (sensetion--synset-id synset)
 		 (sensetion--synset-terms synset)
-		 (sensetion--synset-gloss synset))
+		 (sensetion--synset-def synset))
```
outra coisa é introduzir uma variável `máximo de exemplos a mostrar` e usar isso, e nesse caso faria sentido ter um field gloss como lista com o primeiro elemento sendo definição e o resto exemplos.

se quiser eu implemento amanhã!

odanoburu committed:
* documentation fixes

* use more pcase and cl- functions

odanoburu committed:
* show lexicographer file name in sense menu

(or not)

make it a customizable variable

* remove show of synset id in sense menu

- since that's not useful anymore (we already show terms and lexname)

* limit number of examples shown in sense menu

odanoburu reviewed (commented):
implementei o synset-id novo, e a possibilidade de limitar o número de exemplos. falta testar! tenho uma dúvida sobre a interface que vc pensou: pelo que entendi do código depois que a pessoa escolhe um projeto ela não tem como mudar sem setar a variável na mão, certo?
